FLT/Mathlib/Topology/Algebra/Valued/ValuationTopology.lean

```diff
@@ -8,3 +8,16 @@ lemma Valued.isUnit_valuationSubring_iff {Γ₀ : Type*} [LinearOrderedCommGroup
     IsUnit x ↔ Valued.v (x.val : F) = 1 := by
   convert Valuation.Integers.isUnit_iff_valuation_eq_one _
   exact Valuation.integer.integers _
+
+/-- The unit ball of a valued ring is closed. -/
+theorem Valued.integer_isClosed ( R : Type* ) {Γ₀ : Type*} [Ring R]
+    [LinearOrderedCommGroupWithZero Γ₀] [_i : Valued R Γ₀] : IsClosed (_i.v.integer : Set R) := by
```

Comment:
I think `_i` conventionally means "I will not ever use this variable", so maybe you want to call it `i` or `inst`?